tests/same_window.comlink.test.js

```diff
@@ -579,6 +579,59 @@ describe("Comlink in the same realm", function () {
     expect(() => instance.method()).to.throw();
   });
 
+  it("released proxy should invoke finalizer", async function () {
+    let finalized = false;
+    Comlink.expose(
+      {
+        a: "thing",
+        [Comlink.finalizer]: () => {
+          finalized = true;
+        },
+      },
+      this.port2
+    );
+    const instance = Comlink.wrap(this.port1);
+    expect(await instance.a).to.equal("thing");
+    await instance[Comlink.releaseProxy]();
+    // wait a beat to let the events process
+    await new Promise((resolve) => setTimeout(resolve, 1));
+    expect(finalized).to.be.true;
+  });
+
+  // commented out this test as it could be unreliable in various browsers as
+  // it has to wait for GC to kick in which could happen at any timing
+  // this does seem to work when testing locally
+  it.skip("released proxy via GC should invoke finalizer", async function () {
```

Comment:
Skipped this test, it does work when testing locally but I suspect could prove unreliable in CI.